partner_contact_age_range/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+User can create age ranges.
+When age on concact set it will auto populate age range on contact.
```

Comment:
Please try better, this module will go to OCA. Can't be lazy on readme!
Something like this: 

. Go to Contact / Configuration / Age Ranges
. Configure your Age Ranges
. Contacts will automatically compute ..